tests/foreman/ui_airgun/test_computeresource_ec2.py

```diff
@@ -0,0 +1,190 @@
+
+        1. Create an EC2 compute resource with default properties and taxonomies.
+        2. Update the compute resource name and add new taxonomies.
+        3. Associate compute profile with custom properties to ec2 compute resource
+        4. Delete the compute resource.
+
+    :expectedresults: The EC2 compute resource is created, updated, compute profile associated and
+        deleted.
+
+    :CaseLevel: Integration
+
+    :BZ: 1451626
+
+    :CaseImportance: High
+    """
+    cr_name = gen_string('alpha')
+    new_cr_name = gen_string('alpha')
+    cr_description = gen_string('alpha')
+    new_org = entities.Organization().create()
+    new_loc = entities.Location().create()
+    with session:
+        session.computeresource.create({
+            'name': cr_name,
+            'description': cr_description,
+            'provider': FOREMAN_PROVIDERS['ec2'],
+            'provider_content.access_key': module_ec2_settings['access_key'],
+            'provider_content.secret_key': module_ec2_settings['secret_key'],
+            'provider_content.region.value': module_ec2_settings['region'],
+            'organizations.resources.assigned': [module_org.name],
+            'locations.resources.assigned': [module_loc.name],
+        })
+        cr_values = session.computeresource.read(cr_name)
+        assert cr_values['name'] == cr_name
+        assert cr_values['description'] == cr_description
+        assert (cr_values['organizations']['resources']['assigned']
+                == [module_org.name])
+        assert (cr_values['locations']['resources']['assigned']
+                == [module_loc.name])
+        session.computeresource.edit(cr_name, {
+            'name': new_cr_name,
+            'organizations.resources.assigned': [new_org.name],
+            'locations.resources.assigned': [new_loc.name],
+        })
+        assert not session.computeresource.search(cr_name)
+        cr_values = session.computeresource.read(new_cr_name)
+        assert cr_values['name'] == new_cr_name
+        assert (set(cr_values['organizations']['resources']['assigned'])
+                == {module_org.name, new_org.name})
+        assert (set(cr_values['locations']['resources']['assigned'])
+                == {module_loc.name, new_loc.name})
+        session.computeresource.update_computeprofile(
+            new_cr_name,
+            COMPUTE_PROFILE_LARGE,
+            {
+                'flavor': AWS_EC2_FLAVOR_T2_MICRO,
+                'availability_zone': module_ec2_settings['availability_zone'],
+                'subnet': module_ec2_settings['subnet'],
+                'security_groups.assigned': module_ec2_settings['security_groups'],
+                'managed_ip': module_ec2_settings['managed_ip'],
+            }
+        )
+        session.computeresource.delete(new_cr_name)
+        assert not session.computeresource.search(new_cr_name)
+
+
+@tier2
+def test_positive_create_ec2_with_custom_region(session, module_ec2_settings):
```

Comment:
definitely sounds as tier1 test case, but customer found, hmm, not sure what to do with this